From 1635b15c4ab4a0aabfab3135c01e0112a97ba978 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sun, 8 Apr 2018 19:16:49 +0200
Subject: [PATCH] meson: skip pangoxft headers for the Windows gtk-doc build

The API isn't available on Windows and gtk-doc fails with a
linker error otherwise.

https://bugzilla.gnome.org/show_bug.cgi?id=795045
---
 docs/meson.build | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/docs/meson.build b/docs/meson.build
index c784149e..9122dc3c 100644
--- a/docs/meson.build
+++ b/docs/meson.build
@@ -45,6 +45,11 @@ if host_system != 'windows'
     'pangocairo-win32.h',
     'pangowin32.h',
   ]
+else
+  private_headers += [
+    'pangoxft.h',
+    'pangoxft-render.h',
+  ]
 endif
 
 content_files = [
-- 
2.17.0

